packages/garbo-lib/src/wanderer/lib.ts

```diff
@@ -307,8 +310,12 @@ export function wandererTurnsAvailableToday(
       ? clamp(get(source.property), 0, source.max)
       : 0,
   );
+  // This is the ELG cooldown for spring shoes
+  const freeRun = canWanderCache["freerun"]
+    ? Math.floor(options.estimatedTurns() / 30)
+    : 0;
 
-  return digitize + pigSkinnerRay + yellowRay + wanderers;
+  return digitize + pigSkinnerRay + yellowRay + wanderers + freeRun;
```

Comment:
I don't think we want this. This function is used to determine things like "can we complete this guzzlr quest", and freeRuns aren't used for that. I guess they are for gingerbread city? Ugh I hate that things are different